product_docs/docs/postgis/1.0/01_installing_postgis.mdx

```diff
@@ -25,8 +25,8 @@ The following table lists the latest PostGIS versions and their corresponding Ad
 | `PostGIS 2.3`       | Advanced Server 9.6 and 10             | RHEL/CentOS 7 - x86_64 <br/>RHEL/CentOS 7 - ppc64le <br/>Windows 64 x86 Interactive Installer                                                                                                                              |
 
 !!! Note
-    PostGIS is not supported on CentOS/RHEL/OL 6.x platforms. EDB strongly recommends migrating all EDB products running on these platforms to a supported platform.
-
+    PostGIS is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended that EDB products running on these platforms be migrated to a supported platform.
```

Comment:
I would not change the second sentence. Keep it in active voice.

Reply:
I have changed to maintain consistency as it is "Slony is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended to migrate all EDB products running on these platforms to a supported platform" in all guides.